### ISSUE TITLE
Configure some CPU/memory requests for hub and proxy pods in basehub

### ISSUE THREAD

**Issue by consideRatio** [2i2c-org/infrastructure#2127]
Currently, the hub and proxy pod requests very little CPU/Memory, but various pods by default have already 100m in requests. This could starve our hub/proxy pod of CPU.

It also seems that the hub/proxy pod request of 128 MB memory isn't covering the need. It would be good to request memory more than we typically use so that we don't risk being evicted or OOMKilled.

![image](https://user-images.githubusercontent.com/3837114/216149356-48df6178-4acb-42d4-a090-793c8a945c5e.png)

```
  prod                        proxy-7f5dbcbd68-zqd22                                 10m (0%)      0 (0%)      64Mi (0%)        1Gi (4%)       94d
  prod                        hub-5db4d78fdc-bwbvv                                   10m (0%)      0 (0%)       128Mi (0%)       2Gi (9%)       60m
```

This could have been relevant for the incident in https://github.com/2i2c-org/infrastructure/issues/2126, if it wasn't it would be good to rule it out by having these increased requests.

**Comment by yuvipanda** [comment]
I think looking at observed usage metrics and setting appropriate requests and limits is a good idea! We don't want them to be too high (especially in shared clusters) as that might increase overall cost, but we already have data for this so I leave it to you to figure out a decent number and get it there! I agree that the current situation has to change

**Comment by consideRatio** [comment]
I'm looking at openscapes hub pod, and I observe that the hub responsiveness can peak.

I wonder if that metric including all requests, and if some requests are slow while others fast. The 50th percentile stays low but the 99th is often in seconds.

![image](https://user-images.githubusercontent.com/3837114/218168744-e20c8539-97f0-4fc1-a1c2-2ec908c3408c.png)

**Comment by yuvipanda** [comment]
@consideRatio good catch, I opened https://github.com/jupyterhub/grafana-dashboards/pull/59 as a 'fix' on grafana
